### PR TITLE
feat: set typscript 'Content-Type' header

### PR DESCRIPTION
**Pull request by maximousblk**
This commit sets a custom 'Content-Type' header on '*.ts' files.
We can specify more custom formats if we need to.

I also updated the repository according to template.

### CODE REVIEW

**Review by t8 — APPROVED** [review]
If this has been tested - LGTM!

**Comment by maximousblk** [comment]
I have tested that `*.ts` files now have a `application/javascript` content type. but haven't tested if everything works as expected as there is no defined test to ensure that.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+the issue will likely get closed in favor of issues that have this template.
+Please provide all the info requested in this template.
+
+Please remember that:
+
+  * The person fixing the bug would have to figure out these details anyway. Please be respectful of their time.
+  * You might figure out the issues yourself as you work on extracting these details.
+  * If you list multiple bugs/concerns in this one issue, it makes it hrd to track the progress.
+  * If you open an issue that has many duplicates, the triager may close your issue.
+  * If yuo file something completely blank in the body, the triager may close your issue without further explanation or engagement.
```
Comment (littledivy) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
  * If you file something completely blank in the body, the triager may close your issue without further explanation or engagement.
```

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "tabWidth": 2,
+  "printWidth": 80,
+  "useTabs": false,
+  "proseWrap": "always",
+  "trailingComma": "all",
+  "semi": true,
+  "singleQuote": false,
+  "arrowParens": "always"
+}
```
Comment (littledivy) [inline .prettierrc:10]: THANKS!

**Review by littledivy — CHANGES_REQUESTED** [review]
Overall LGTM! Thanks

**Comment by t8** [comment]
@justablob would you mind giving this a review?

**Review by justablob — APPROVED** [review]
LGTM

**Pull request merged**